src/Examples.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Link, RouteComponentProps } from 'react-router-dom';
+
+import * as Snippet from "./Snippet";
+
+import * as Simple from "./examples/Simple";
+
+interface IExample {
+    lang: string;
+    code: string;
+}
+
+const sources = {
+  "simple": Simple,
+};
+
+function ExampleNames() {
+  return Object.keys(sources);
+}
+
+function ParseExample(ex: IExample): Snippet.ISnippetProps {
+    return {
+        blocks: [{code: "def main():", good: false}] as Snippet.IBlock[],
+        lang: "python",
+    };
+}
+
+function GetExample(name: string) {
+    return ParseExample(sources[name]);
+}
+
+class Example extends React.Component<RouteComponentProps<any>, {}> {
```

Comment:
I'm guessing `RouteComponentProps<any>` is because it's still prototyping? I.e., we'll have a more explicit interface for ExampleProps later?

Reply:
That was the best thing I could figure out for typescript to tell me RouteComponentProps didn't have the properties I need, but I'll try to fiddle more and figure something out.